CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### 0.14.0
+
+* Update to be built on top of package:observable
```

Comment:
if it's not too hard, maybe add a note about what kind of breaking changes there are?